src/NServiceBus.AcceptanceTests/ConfigureEndpointMsmqTransport.cs

```diff
@@ -5,15 +5,20 @@
 using System.Threading.Tasks;
 using NServiceBus;
 using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.AcceptanceTests.ScenarioDescriptors;
 using NServiceBus.Configuration.AdvanceExtensibility;
 using NServiceBus.Transport;
 
 public class ConfigureEndpointMsmqTransport : IConfigureEndpointTestExecution
 {
+    const string DefaultConnectionString = "cacheSendConnection=false;journal=false;";
+
     public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
     {
         queueBindings = configuration.GetSettings().Get<QueueBindings>();
-        var connectionString = settings.Get<string>("Transport.ConnectionString");
+        var connectionString =
+            EnvironmentHelper.GetEnvironmentVariable($"{nameof(ConfigureEndpointMsmqTransport)}.ConnectionString")
```

Comment:
Is this correct? Isn't the convention we use

MsmqTransport.ConnectionString ?

so essentially

`EnvironmentHelper.GetEnvironmentVariable($"{nameof(MsmqTransport)}.ConnectionString")`